docs/pulsar_geo_replication.md

```diff
@@ -0,0 +1,185 @@
+```
+
+### Create a Destination PulsarConnection
+First, you need to create a destination PulsarConnection, this connection indicates the info of the remote pulsar cluster.
+You need to add two extra fields `clusterName` and `brokerServiceURL` for this PulsarConnection.
+
+Notice, the token of the authentication only support using `value` now.
+
+```yaml
+apiVersion: resource.streamnative.io/v1alpha1
+kind: PulsarConnection
+metadata:
+  name: cluster1-dest-connection
+  namespace: cluster1
+spec:
+  clusterName: cluster2-sn-platform # the remote cluster name
+  adminServiceURL: http://cluster2-sn-platform-broker.cluster2.svc.cluster.local:8080 # the remote pulsar admin service
+  brokerServiceURL: pulsar://cluster2-sn-platform-broker.cluster2.svc.cluster.local:6650 # the remote pulsar broker service
+  authentication:
+    token:
+      value: YzBkMzVjOGEtNWRiNi0yZGM3LTI4NjktZTM1NGU1YmJiNTY4OjAwZTk2MzgzLWFjY2YtMGMzOC1kNmRlLTY1ODc0ODkxNmE1ZQo=
+
+```
+
+
+### Create a PulsarGeoReplication
+It means setup the geo replication from the `cluster1` to `cluster2`. The operator will create a new cluster called `cluster2-sn-platform`.
+
+```yaml
+apiVersion: resource.streamnative.io/v1alpha1
+kind: PulsarGeoReplication
+metadata:
+  name: cluster1-geo
+  namespace: cluster1
+spec:
+  connectionRef:
+    name: cluster1-connection
+  destinationConnectionRef:
+    name: cluster1-dest-connection
+  lifecyclePolicy: CleanUpAfterDeletion
+```
+
+### Grant the permission for the tenant
+
+You can create a new tenant or update the existing the tenant with the field `geoReplicationRefs`. It will assign the `cluster2-sn-platform` to the tenant.
+
+```yaml
+apiVersion: resource.streamnative.io/v1alpha1
+kind: PulsarTenant
+metadata:
+  name: cluster1-geo-tenant
+  namespace: cluster1
+spec:
+  name: geo-test
+  connectionRef:
+    name: cluster1-connection
+  geoReplicationRefs:
+  - name: cluster1-geo
+  lifecyclePolicy: CleanUpAfterDeletion
+```
+
+### Enable the geo replication at namespace level
+
+You can create a new namespace or update the existing the namespace with the field `geoReplicationRefs`. It will assign the namespace to `cluster2-sn-platform`.
+
+Notice, once you create a geo-replication namespace, any topics that producers or consumers create within that namespace are replicated across clusters
+
+```yaml
+apiVersion: resource.streamnative.io/v1alpha1
+kind: PulsarNamespace
+metadata:
+  name: cluster1-geo-namespace
+  namespace: cluster1
+spec:
+  name: geo-test/testn1
+  geoReplicationRefs:
+  - name: cluster1-geo
+  connectionRef:
+    name: cluster1-connection
+  backlogQuotaLimitSize: 1Gi
+  backlogQuotaLimitTime: 24h
+  bundles: 16
+  messageTTL: 1h
+  lifecyclePolicy: CleanUpAfterDeletion
+
+```
+
+
+### Enable the geo replication at topic level
+
+You can create a new topic or update the existing the topic with the field `geoReplicationRefs`. It will assign the topic to `cluster2-sn-platform`.
```

Comment:
```suggestion
You can create a new topic or update an existing topic by adding the field `geoReplicationRefs`. It will add the topic to `cluster2-sn-platform`.
```